src/components/Global/Analytics/TopPools.tsx

```diff
@@ -0,0 +1,226 @@
+import { useContext, useEffect } from 'react';
+import { CrocEnvContext } from '../../../contexts/CrocEnvContext';
+import useFetchPoolStats from '../../../App/hooks/useFetchPoolStats';
+import { topPoolIF } from '../../../App/hooks/useTopPools';
+import TokenIcon from '../TokenIcon/TokenIcon';
+import uriToHttp from '../../../utils/functions/uriToHttp';
+import { useNavigate } from 'react-router-dom';
+type PoolStats = {
+    poolName?: string;
+    baseLogoUri?: string;
+    quoteLogoUri?: string;
+    poolVolume?: string;
+    poolPrice?: string;
+    poolTvl?: string;
+    poolApy?: string;
+    poolPriceChangePercent?: string;
+    isPoolPriceChangePositive?: boolean;
+
+    baseTokenCharacter?: string;
+    quoteTokenCharacter?: string;
+};
+export default function TopPools() {
+    const { topPools } = useContext(CrocEnvContext);
+
+    const formattedPoolData: PoolStats[] = [];
+    useEffect(() => {
+        const fetchPoolData = async () => {
+            for (const pool of topPools) {
+                const poolStats = await useFetchPoolStats(pool);
+                formattedPoolData.push(poolStats);
+            }
+        };
+
+        fetchPoolData();
+    }, []);
+
+    const PoolRow = (pool: topPoolIF) => {
+        const poolData = useFetchPoolStats(pool);
+        const {
+            poolName,
+            baseLogoUri,
+            quoteLogoUri,
+            poolVolume,
+            poolPrice,
+            poolTvl,
+            poolApy,
+            poolPriceChangePercent,
+            isPoolPriceChangePositive,
+            poolLink,
+        } = poolData;
+
+        const navigate = useNavigate();
+
+        const navigateToTrade = () => {
+            navigate(poolLink);
+        };
+
+        return (
+            <tr
+                className='hover:bg-dark2 cursor-pointer'
+                onClick={navigateToTrade}
+            >
+                <td className='px-6  whitespace-nowrap'>
+                    <div className='flex items-center'>
+                        <div className='flex-shrink-0 flex items-center'>
+                            <TokenIcon
+                                src={uriToHttp(baseLogoUri ?? '...')}
+                                alt={poolName}
+                                size='2xl'
+                            />
+                            <TokenIcon
+                                src={uriToHttp(quoteLogoUri ?? '...')}
+                                alt={poolName}
+                                size='2xl'
+                            />
+                        </div>
+                        <div className='ml-4  sm:hidden md:hidden lg:hidden xl:hidden'>
```

Comment:
what does the hidden do, and why is it all hidden?

Reply:
On smaller devices, the pool name is added to the token pair logos. We do not need it anywhere else, hence hidden:
![image](https://github.com/CrocSwap/ambient-ts-app/assets/83131501/955826d7-d6b5-4f70-bbcf-47e8d1fe831e)